From b7b53d7cd1f8f83dc1e5629e5f7445c0cde2f91b Mon Sep 17 00:00:00 2001
From: Michael Ferguson <mfergs7@gmail.com>
Date: Fri, 13 Mar 2015 01:00:30 -0700
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2b4085c..03e55aa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,6 +58,7 @@ install(DIRECTORY src/include/sbpl DESTINATION include
 	FILES_MATCHING PATTERN "*.h")
 install(TARGETS sbpl
 	LIBRARY DESTINATION lib)
+install(FILES package.xml DESTINATION share/sbpl)
 
 add_executable(test_sbpl src/test/main.cpp)
 target_link_libraries(test_sbpl sbpl)
-- 
1.9.1

